app/javascript/packs/views/restrooms/index.js

```diff
@@ -1,50 +1,40 @@
 import { Maps } from '../../lib/maps';
 
-$(function(){
-  var headerHidden = false;
-  var mapInitialized = false;
-  var mapShow = false;
-  var mapContainer = $("#mapContainer");
-  var list = $("#list");
-  var mapToggle = $(".mapToggle");
-  var search = $("#search");
+let URLParamsParser = window.URLSearchParams;
 
-  function toggleMap (){
-    if (mapShow) {
-      mapToggle.html("Map View");
-      // animate
-      mapContainer.fadeOut(500, function() { list.fadeIn(500) });
-    } else{
-      mapToggle.html("List View");
-      // animate
-      list.fadeOut(500, function() { mapContainer.fadeIn(500, initPoints) });
-    }
-    mapShow = !mapShow;
+// Polyfill for Poltergeist.
```

Comment:
We do hope to support most older browsers.

I guess Poltergeist is the one that needs to work for testing purposes... but this comment can probably say "Polyfill for older browsers"?

Reply:
Yes @DeeDeeG you're right!
I'll change this to `old browsers` as it fits better!

Thanks!

Reply:
An update on the review process: I'm hoping to get the rubocop pull request (#644) merged first thing, then come back to pull requests such as this one.

Reply:
Sure @DeeDeeG ! No rush!